configure

```diff
@@ -5583,6 +5594,25 @@ if test "$_pandocext" = "default"; then
 	fi
 fi
 
+#
+# Check for Discord
+#
+echocheck "Discord RPC"
+if test "$_discord" = auto ; then
+	_discord=no
+	cat > $TMPC << EOF
+#include <discord_rpc.h>
+int main(void) { Discord_Shutdown(); return 0; }
+EOF
+	cc_check $DISCORD_CFLAGS $DISCORD_LIBS -ldiscord-rpc && _discord=yes
```

Comment:
Do you require some specific minimal version of the library?

Reply:
No.